13/umbraco-cms/tutorials/getting-started-with-entity-framework-core.md

```diff
@@ -96,7 +96,10 @@ We need to register the `DbContext` to be able to use it in Umbraco.
 To do this we can use this helpful extension method:
 
 ```csharp
-services.AddUmbracoEFCoreContext<BlogContext>("{YOUR CONNECTIONSTRING HERE}", "{YOUR PROVIDER NAME HERE}");
+builder.Services.AddUmbracoDbContext<BlogContext>(options => 
+{
+    options.UseSqlServer("{YOUR CONNECTIONSTRING HERE}");
+});
```

Comment:
```suggestion
builder.Services.AddUmbracoDbContext<BlogContext>(options => 
{
    options.UseSqlServer("{YOUR CONNECTIONSTRING HERE}");
    //If you are using SQlite, replace UseSqlServer with UseSqlite
});
```

Reply:
this seems to be repeating twice so in this case my suggestion would be to remove this above example 🤔